tests/e2e_tests/Dockerfile.gui

```diff
@@ -0,0 +1,23 @@
+FROM python:2
```

Comment:
Is there any hard requirement for Python 2? The `test_ui.py` claims to be compatible with Python 3

Reply:
the `test_ui.py` file should be, but the `helpers.py` and some more transitive requirements also require fabric - the changes in the fabric api (to v2 or better v3) would probably require quite a few more changes to a range of tests..

Reply:
Okey, I understood now, thanks. I didn't realize the dependency on MenderAPI and helpers.

Yes, we have already tasks in QA project to migrate to Fabric >2 (and then Python 3), so we don't need to do them now.